pkgs/development/python-modules/inflate64/default.nix

```
@@ -0,0 +1,35 @@
+{
+  buildPythonPackage,
+  fetchFromGitea,
+  lib,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.1";
+
+  pyproject = true;
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "miurahr";
+    repo = "inflate64";
+    rev = "v${version}";
+    hash = "sha256-deFx8NMbGLP51CdNvmZ25LQ5FLPBb1PB3QhGhIfTMfc=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
As mentioned already.